pkg/vulnsrc/vulnsrc.go

```diff
@@ -47,15 +49,31 @@ func Update(names []string) error {
 
 	// Clone vuln-list repository
 	dir := filepath.Join(utils.CacheDir(), "vuln-list")
-	updatedFiles, err := git.CloneOrPull(repoURL, dir)
-	if err != nil {
-		return xerrors.Errorf("error in vulnsrc clone or pull: %w", err)
-	}
-	log.Logger.Debugf("total updated files: %d", len(updatedFiles))
+	//updatedFiles, err := git.CloneOrPull(repoURL, dir)
+	//if err != nil {
+	//	return xerrors.Errorf("error in vulnsrc clone or pull: %w", err)
+	//}
+	//log.Logger.Debugf("total updated files: %d", len(updatedFiles))
+	//
+	//// Only last_updated.json
+	//if len(updatedFiles) <= 1 {
+	//	return nil
+	//}
 
-	// Only last_updated.json
-	if len(updatedFiles) <= 1 {
+	updatedFiles := map[string]struct{}{}
+	err := filepath.Walk(dir, func(path string, info os.FileInfo, err error) error {
+		if info.IsDir() {
```

Comment:
This currently panics if info is nil (which it can be if the cache doesn't exist yet)

Reply:
Hmm... we should handle err from this args before `info.IsDir()`.